src/OneMightyRoar/PHP_ActiveRecord_Components/AbstractModel.php

```diff
@@ -523,23 +523,21 @@ public static function buildPagingOptions(array $paging_options, $page = null)
 
         $per_page   = isset($paging_options['per_page']) ? (int) $paging_options['per_page'] : static::DEFAULT_LIMIT;
 
+        $quoted_table_name = static::connection()->quote_name(static::table_name());
+        $primary_keys = static::table()->pk;
+        $primary_primary_key = isset($primary_keys[0]) ? $primary_keys[0] : null;
```

Comment:
I thought about that, but I didn't want to change the current behavior. We could do a follow up change, but I'd be weary considering the potential effects of this change already.